corehq/apps/registry/models.py

```diff
@@ -26,6 +27,17 @@ def owned_by_domain(self, domain, is_active=None):
             query = query.filter(is_active=is_active)
         return query
 
+    def visible_to_domain(self, domain):
```

Comment:
When would you want to do `visible_to_domain(domain)` vs `accessible_to_domain(domain)`?  Why have both?  Looks like the difference is that the latter will exclude a domain's own registries unless its invited.  Are projects ever, sometimes, or never invited to their own registries?

Reply:
The distinction comes in when you want to access the data in a registry. In order opt in / out of the registry you need to be able to 'see' it in the UI so that's 'visible_to_domain'. This is also likely what will get used in app manager / UCRs when configuring case search etc.

If you want to query the registry then you need to have accepted the invitation and the registry must be 'active' and that's covered by 'accessible_to_domain'. 

If you have better naming ideas I'm all ears. I'll add a bit more to the doc string as well.

Reply:
no that makes sense, thanks